docs/UserGuide.md

```diff
@@ -283,7 +283,7 @@ Format: `deleteEvent INDEX EVENT_NUMBER`
 
 * `INDEX` refers to the index number shown in the displayed person list, whereas `EVENT_NUMBER` refers to the event's index shown in the person's full schedule. Both indices **must be positive integers** 1, 2, 3, …​
 
-Example:
+Examples:
```

Comment:
Em there is only 1 example here

Reply:
I was thinking we standardise and put everything plural